### PR TITLE
Shwethac/add pipelinefile

### PR DESCRIPTION
**Pull request by schandra13**
Adding in azure_pipeline.yml to point to release.yml in jdk-harness in Devdiv

### CODE REVIEW

File: .azure-pipelines/azure_devops.yml [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+      ref: refs/heads/main
+      project: "Devdiv"
+
+variables:
+  JTREG_VERSION: "6.1"
+
+stages:
+  - template: /.devops/jdk11u/release.yml@jdk-harness
+
+name: "$(Year:yyyy).$(Month).$(BuildID)-ci"
```
Comment (joe-braley) [inline .azure-pipelines/azure_devops.yml:25]: What is the purpose of keeping L25 - L46?

Are there any conflicts with using multiple triggers?

File: .azure-pipelines/azure_devops.yml [hunk 2]
```diff
@@ -1,22 +1,47 @@
...
+    include:
+      - main
+
+pr: none
+
+resources:
+  repositories:
+    - repository: jdk-harness
+      type: git
+      name: Devdiv/jdk-harness
```
Comment (joe-braley) [inline .azure-pipelines/azure_devops.yml:15]: Q: Still require a `name:` field when specifying a `repository:` and `project:` ? 

(Could be a best practice to include anyways?)

File: .azure-pipelines/azure_devops.yml (hunk 1 above)
Comment (joe-braley) [inline .azure-pipelines/azure_devops.yml:25]: Q: Purpose of multiple names? Here on L25 and L1.

File: .azure-pipelines/azure_devops.yml [hunk 3]
```diff
@@ -1,22 +1,47 @@
...
+pr: none
+
+resources:
+  repositories:
+    - repository: openjdk_azure_devops_harness
+      type: git
+      name: github_move/openjdk-azure-devops-harness
+      ref: refs/heads/main
+
+variables:
```
Comment (joe-braley) [inline .azure-pipelines/azure_devops.yml:42]: Nit: Duplicate variables.

File: .azure-pipelines/azure_devops.yml [hunk 4]
```diff
@@ -1,22 +1,47 @@
...
+  batch: true
+  branches:
+    include:
+      - main
+
+pr: none
+
+resources:
+  repositories:
+    - repository: openjdk_azure_devops_harness
```
Comment (joe-braley) [inline .azure-pipelines/azure_devops.yml:37]: Q: Should we keep this resource? Think it was used for demo purposes.

**Review by joe-braley — CHANGES_REQUESTED** [review]
Couple questions.

File: .azure-pipelines/azure_devops.yml (hunk 1 above)
Comment (schandra13) [reply .azure-pipelines/azure_devops.yml:25]: I think those are from the original file from patches
I do not have these changes in my local

File: .azure-pipelines/azure_devops.yml (hunk 4 above)
Comment (schandra13) [reply .azure-pipelines/azure_devops.yml]: Removed the lines from demo

File: .azure-pipelines/azure_devops.yml (hunk 3 above)
Comment (schandra13) [reply .azure-pipelines/azure_devops.yml]: Removed those extra lines

File: .azure-pipelines/azure_devops.yml (hunk 2 above)
Comment (schandra13) [reply .azure-pipelines/azure_devops.yml:15]: All the fields would reference to Devdiv jdk-harness. Originally it worked without having project: but was throwing errors when I was testing it out.

**Review by joe-braley — APPROVED** [review]
LGTM

**Pull request merged**